src/main.rs

```diff
@@ -19,64 +21,67 @@ use cargo_binstall::{
     *,
 };
 
-#[derive(Debug, StructOpt)]
+static DEFAULT_TARGET: OnceCell<String> = OnceCell::new();
+
+#[derive(Debug, Parser)]
+#[clap(version, about = "Install a Rust binary... from binaries!")]
 struct Options {
     /// Package name or URL for installation
     /// This must be either a crates.io package name or github or gitlab url
-    #[structopt()]
+    #[clap(value_name = "crate")]
     name: String,
 
     /// Filter for package version to install, in Cargo.toml format.
     /// Use `=1.2.3` to install a specific version.
-    #[structopt(long, default_value = "*")]
+    #[clap(long, default_value = "*")]
     version: String,
 
     /// Override binary target, ignoring compiled version
-    #[structopt(long, default_value = TARGET)]
+    #[clap(help_heading = "OVERRIDES", long, default_value = DEFAULT_TARGET.get().unwrap())]
     target: String,
```

Comment:
Since #161 introduces `detect_targets` that returns multiple targets, would it be a good idea to also use `Vec<Box<str>>` here and parses the target using `arg.split(',')`?